src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  width: $card-width;
+
+  padding: 32px 16px 16px;
+
+  border: 1px solid $card-border-color; // #f3f3f3
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+
+  transition-duration: $anim-duration;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    align-self: center;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    color: $main-color;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    transition-duration: inherit;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    color: $secondary-color;
+
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    margin-top: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
```

Comment:
it's not a good practice, better use this syntax:

```
  &__rating {
    margin-top: 16px;

    display: flex;
    justify-content: space-between;
    align-items: center;
  }
  
 &__rating-text {
    font-size: 10px;
    line-height: 14px;
  }
```
It will increase readability